components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -4710,6 +4729,15 @@ private ServiceProviderProperty buildIsManagementAppProperty(ServiceProvider sp)
         return isManagementAppProperty;
     }
 
+    private ServiceProviderProperty buildIsB2BSSAppPropertyProperty(ServiceProvider sp) {
```

Comment:
Does having `Property` twice in the method name intended?

Reply:
Nop, refactored the methodname